pkgs/development/compilers/factor-lang/factor99.nix

```diff
@@ -219,6 +214,5 @@ stdenv.mkDerivation {
     license = licenses.bsd2;
     maintainers = with maintainers; [ vrthra spacefrogg ];
     platforms = lib.intersectLists platforms.x86_64 platforms.linux;
-    mainProgram = "factor";
```

Comment:
"factor" is still the main program.

Reply:
I must have somehow cloned a nixpkgs state that didn't include your commit. Of course these snippets should not be removed.